core/domain/rights_manager.py

```diff
@@ -628,6 +628,16 @@ def _publish_activity(committer_id, activity_id, activity_type):
         committer_id, activity_id, activity_type, ACTIVITY_STATUS_PUBLIC,
         '%s published.' % activity_type)
 
+    #Add contribution dates to the owners and editors
```

Comment:
This is going to also include collections...

@BenHenning -- what do you think? Should the rules for determining the first contribution datetime be the same for collections as for explorations? I.e. you publish a collection, or a collection you contribute to gets published, or you edit a published collection